main.tf

```diff
@@ -294,7 +294,7 @@ resource "aws_cloudwatch_log_group" "forwarder_log_group" {
 # Scheduled retry
 
 resource "aws_iam_role" "scheduled_retry" {
-  count = var.dd_store_failed_events && var.dd_schedule_retry_failed_events ? 1 : 0
+  count = coalesce(var.dd_store_failed_events, false) && coalesce(var.dd_schedule_retry_failed_events, false) ? 1 : 0
```

Comment:
drive-by fix, this errors if both variables are not provided `Error during operation: argument must not be null.`